bool door
int64 linActs_speed
int64 arm_speed
int64 drum_speed